exchanges/account/account_test.go

```diff
@@ -337,20 +318,16 @@ func TestGetFree(t *testing.T) {
 	}
 }
 
-func TestUpdate(t *testing.T) {
+func TestLoad(t *testing.T) {
```

Comment:
```suggestion
func TestSave(t *testing.T) {
```

Since its name has been updated